### ISSUE TITLE
sf_describe_object_fields() fails for some objects

### ISSUE THREAD

**Issue by AleksanderPolitanski** [StevenMMortimer/salesforcer#27]
Hello,
I'm getting an error when running sf_describe_object_fields("Object__c") for some objects. 
The error message is:
"Error: All columns in a tibble must be 1d or 2d objects:
Column `validFor` is NULL"

For example this error occurs for "Account" and "Opportunity" objects 
but not for "Event" or "Task" objects in my org.  
Would you be able to give me any hints?
Maybe this error is already knowned? 

Many thanks in advance

**Comment by StevenMMortimer** [comment]
@AleksanderPolitanski There is a similar issue and discussion that was documented in #16. The short answer is that sometimes organizations have custom objects or fields that are deeply nested or something that is not understood by the parsing function. I cannot help you unless you send me the output of a function that describes the object because I do not know how your org is structured. If you run the following commands and email me the output (reportmort@gmail.com) then I'd be happy to take a look. **Note** that these functions will only give me the structure of the object in your org and not the records from it, so it should not contain any identifiable record information that you might not want to share with me, but it will tell me the field names and picklist values of the object. If that information is too sensitive to share with me, then you will have to debug on your own. 

**Function to create data for me to debug (send the RDS file to reportmort@gmail.com)**

```
library(salesforcer)
sf_auth()

obj_dat <- sf_describe_objects(object_names = 'Object__c', api_type = "SOAP")[[1]]
acct_dat <- sf_describe_objects(object_names = 'Account', api_type = "SOAP")[[1]]
oppty_dat <- sf_describe_objects(object_names = 'Opportunity', api_type = "SOAP")[[1]]

saveRDS(obj_dat, "obj_dat.rds")
saveRDS(acct_dat, "acct_dat.rds")
saveRDS(oppty_dat, "oppty_dat.rds")
```

**Comment by StevenMMortimer** [comment]
In some Salesforce objects the picklists have a "validFor" field that is NULL for some picklist options. I did not expect NULLs to be there. This was fixed in 60b9d5de9d4ead30e5072140a6a44c778669a19b.
